cmd/helm/repo_remove_test.go

```diff
@@ -95,4 +81,74 @@ func TestRepoRemove(t *testing.T) {
 	if f.Has(testRepoName) {
 		t.Errorf("%s was not successfully removed from repositories list", testRepoName)
 	}
+
+	// Test removal of multiple repos in one go
+	var testRepoNames = []string{"foo", "bar", "baz"}
+	b2 := bytes.NewBuffer(nil)
```

Comment:
Is it right to use the same buffer object for all three repos?

Reply:
The buffer is simply used to gather the printouts of the remove command.  You can use the same buffer throughout the function as long as you call `b.Reset()` before calling a new `repo remove` command.